website/docs/reference/resource-configs/grants.md

```diff
@@ -98,152 +72,117 @@ The `meta` config can also be defined under the `seeds` config block in `dbt_pro
 <File name='snapshots/schema.yml'>
 
 ```yml
-version: 2
-
 snapshots:
   - name: snapshot_name
-    config:
-      meta: {<dictionary>}
-
-    columns:
-      - name: column_name
-        meta: {<dictionary>}
-
+    config:  
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined:
+The `grants` config can also be defined:
+
 - under the `snapshots` config block in `dbt_project.yml`
 - in a `config()` Jinja macro within a snapshot's SQL block
 
 See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
+</Tabs>
 
-<TabItem value="analyses">
-
-The `meta` config is not currently supported for analyses.
-
-</TabItem>
+## Definition
 
-<TabItem value="macros">
+You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt.
 
-<File name='macros/schema.yml'>
+## Database-specific requirements and notes
 
-```yml
-version: 2
+While we try to standardize the terms we use to describe different features, you will always find nuances in different databases. This section outlines some of those database-specific requirements and notes.
 
-macros:
-  - name: macro_name
-    meta: {<dictionary>}
+### Common syntax 
 
-    arguments:
-      - name: argument_name
-        meta: {<dictionary>}
+In our examples, you find terms like `select` and `another_user` because many databases use these terms, but be aware of the syntax your own database supports:
 
-```
+* Privileges: A right to perform an action in a database.
+* Grantees: A way to manage privileges. Recipients of granted privileges, also called "principals." Grantees can be a user, a group of users, a role held by users (Snowflake), a service account (GCP), and more.
 
-</File>
+<WHCode>
 
-</TabItem>
+<div warehouse="BigQuery">
 
-<TabItem value="exposures">
+- Be aware that the `grants` config is unrelated to the `grant_access_to` config:
```

Comment:
This is great! Let's add a note to this same effect in the docs for `grant_access_to`. Perhaps a snippet / FAQ in both spots?

Reply:
Great idea! Addressed here: https://github.com/dbt-labs/docs.getdbt.com/pull/1642/commits/d05223dfa7c8c91717bc8d0609b18556a5a91edf